README.md

```diff
@@ -29,14 +29,14 @@ developer communities.
 * Unity environment control from Python
 * 15+ sample Unity environments
 * Two deep reinforcement learning algorithms,
-[Proximal Policy Optimization](https://github.com/Unity-Technologies/ml-agents/tree/latest_release/docs/Training-PPO.md)
```

Comment:
Can we keep these pointing to `latest_release`?

Reply:
Why? Is there a reason for it?

Reply:
So that people landing on the main page will get linked to the latest_release of the docs instead of the master docs (which are sometime incompatbiel)

Reply:
This would mean that we would need to change it everytime we do a release right? (since we do not was any latest_release tag in the release branches.

Reply:
That would only happen if they are on the master branch (in which case the docs should point to master). If they want the docs for the latest release, they should actually check the tag for latest_release.

I think you are pointing to another issue, which is latest_release branch should be the landing page of ml-agents.

Reply:
OK, let's take the discussion offline. This is good to merge as it is.